README.md

```
@@ -105,68 +117,190 @@ Meet our wonderful team comprised of Product Managers, Designers, Tech Leads, En
 
-## Getting Started
+## 🛠️ Project Setup
+
+This section is pretty important, and messing up here could lead to a lot of issues later in the development process. If you aren't sure about any of these directions or something isn't working for you please reach out to one of the Tech Leads!
+
+Install the following software from the links provided:
+- [Git](https://git-scm.com/)
+- [Node.js **v22**](https://nodejs.org/)
 
-First, run the development server:
+It is recommended that you use VSCode as your IDE. You can install it [here](https://code.visualstudio.com/) if you don't have it already. 
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)